### ISSUE TITLE
I got bad results when reproducing 

### ISSUE THREAD

**Issue by parallelarc** [hisfog/SfMNeXt-Impl#16]
Nice work! But I got some problems when reproducing. That is my config. 
`--data_path ../raw_data/
--log_dir ./logdir/
--model_name res_640x192
--eval_split eigen
--backbone resnet_lite
--height 192
--width 640
--batch_size 16
--num_epochs 25
--scheduler_step_size 15
--num_layers 50
--num_features 256
--model_dim 32
--patch_size 16
--dim_out 64
--query_nums 64
--eval_mono
--post_process
--load_pretrained_model
--load_pt_folder ./pretrained/
--pretrained_pose
--pose_net_path ./pretrained/` 

I download pretrained model (depth.pth, encoder.pth and pose.pth) in the "pretrained" folder and I use jpeg images to train my model. That is what I got. 

![2](https://github.com/hisfog/SfMNeXt-Impl/assets/63178075/827152b5-3f00-42a9-9407-9d2305c49615))
![1](https://github.com/hisfog/SfMNeXt-Impl/assets/63178075/4f652b48-9174-4d22-a2b8-44e6e02f320a)

**Comment by parallelarc** [comment]
> It seems that you did not normalize the depth map, which means you saved the raw depth values. For visualization, you should normalize the depth map to [0, 1] and then save it as an image.

Thanks! I will have a try.
